languages/csharp/exercises/concept/namespaces/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+Namespaces are a way to group related code and to avoid name clashes and are generally present in all but the most trivial code base.
+
+The syntax is as follows:
+
+```csharp
+namespace MyNameSpace
+{
+    public class MyClass {}
+
+    public class OtherClass {}
+}
+```
+
+Types enclosed in namespaces are referred to outside the scope of the namespace by prefixing the type name with the dot syntax. Alternatively, and more usually, you can place a `using` directive at the top of the file (or within a namespace) and type can be used without the prefix. Within the same namespace there is no need to qualify type names.
+
+```csharp
+namespace MySpace
```

Comment:
This is very nice! Did you consider using this example in the `after.md` document too?

Reply:
Good idea.  Done